Jenkinsfile

```diff
@@ -333,7 +333,7 @@ pipeline {
           axis {
             // the list of support java versions can be found in the infra repo (ansible/roles/java/defaults/main.yml)
             name 'JAVA_VERSION'
-            values 'openjdk12', 'openjdk13', 'openjdk14', 'openjdk17'
+            values 'openjdk17', 'openjdk18'
```

Comment:
is 18 available? I think it's pre-release

Reply:
Yes, it is available in the infra repo, but I haven't tested it so far, this PR will let us know for sure.

Reply:
Well, the good news is that OpenJDK 18 is available, the bad (and somehow expected) is that it fails.

For example Jacoco seems to fail due to this https://github.com/jacoco/jacoco/pull/1198, which is merged but not released, should be in version 0.8.8. Thus I'll disable it for now and create a separate issue to track all the tasks to support JDK 18.

Reply:
https://github.com/elastic/apm-agent-java/issues/2328 is the new shopping list for Java 18 compatibility when building the project.